src/createServer.js

```diff
@@ -1,3 +1,32 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { normalizeUrl } = require('./utils/normalizeUrl');
+const { getErrors } = require('./utils/getErrors');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { sendRequest } = require('./utils/sendRequest');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [textToFormat, caseToFormat] = normalizeUrl(req);
+    const errors = getErrors(textToFormat, caseToFormat);
+
+    if (errors.length > 0) {
+      sendRequest(res, 'Bad request', 400, { errors });
+    } else {
+      const { originalCase, convertedText } = convertToCase(
+        textToFormat,
+        caseToFormat,
+      );
+
+      sendRequest(res, 'OK', 200, {
+        originalCase,
+        convertedText,
+        targetCase: caseToFormat,
+        originalText: textToFormat,
+      });
```

Comment:
move it to the variable for better readable
```suggestion
      sendRequest(res, 'OK', 200, ...);
```